### ISSUE TITLE
Change tint of selected range?

### ISSUE THREAD

**Issue by JStrawn** [yonat/MultiSlider#13]
Thank you so much for making this library!

Is there currently any way you can think of to change the color for the highlighted range? Here's an example of what I mean:

![image](https://user-images.githubusercontent.com/24421009/45703662-fce67f00-bb42-11e8-9a55-1a395f2c3a0e.png)

**Comment by yonat** [comment]
I haven’t implemented that, but you’re welcome to submit a pull request!

**Comment by omerelimelech** [comment]
i guess its not the most efficient way of doing that,
but it works great with 1 or 2 thumbs.

enjoy :)

```
let selectedAreaColor = UIColor.blue.cgColor
    let nonSelectedAreaColor = UIColor.lightGray.cgColor
    @objc func updateValues(_ slider: MultiSlider){
        slider.tintColor = .clear
        slider.trackView.layer.sublayers?.forEach({ (subLayer) in
            if subLayer is CAGradientLayer{
                subLayer.removeFromSuperlayer()
            }
        })
        let layer: CAGradientLayer = CAGradientLayer()
        layer.frame.size = CGSize(width: slider.frame.width , height: slider.trackView.frame.height)
        layer.startPoint = CGPoint.zero
        layer.endPoint = CGPoint(x: 1, y: 0)
        layer.colors = [
            selectedAreaColor,
            selectedAreaColor,
            nonSelectedAreaColor,
            nonSelectedAreaColor,
            selectedAreaColor,
            selectedAreaColor
        ]
        let diff = slider.maximumValue - slider.minimumValue
        var arr = [0.0] as [NSNumber]
        slider.value.forEach { (value) in
            let side1 = (value - slider.minimumValue) / diff as NSNumber
            let side2 = ((value - slider.minimumValue) / diff) / 2 as NSNumber
            arr.append(side1)
            arr.append(side2)
        }
        layer.locations = arr
        if slider.hasRoundTrackEnds{
            layer.cornerRadius = slider.trackView.frame.height / 2
        }
        slider.trackView.layer.insertSublayer(layer, at: 0)
```

**Comment by yonat** [comment]
implemented in 1.9.0.
